lib/t/accounts.ex

```diff
@@ -521,8 +521,17 @@ defmodule T.Accounts do
 
       {:ok, nil}
     end)
-    |> Multi.run(:show_profile, fn _repo, %{user: user} ->
+    |> Multi.run(:show_profile, fn repo, %{user: user} ->
       {count, nil} = maybe_unhide_profile_with_story(user.id)
+
+      # TODO move someplace else
+      if count >= 1 do
+        User
+        |> where(id: ^user.id)
+        |> update([u], set: [onboarded_with_story_at: fragment("now()")])
+        |> repo.update_all([])
+      end
+
```

Comment:
not nice, but it gets the job done, sets `onboarded_with_story_at` first time the profile's `hidden?` toggles